vars/withFortifySecrets.groovy

```diff
@@ -0,0 +1,15 @@
+def call(String product, String environment, String fortifyVaultName, Closure block) {
```

Comment:
with https://github.com/hmcts/cnp-jenkins-library/pull/755/files#r665241989 this becomes
```suggestion
def call(String fortifyVaultName, Closure block) {
```

Reply:
done